src/Repository/RepositoryStack.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Repository;
+
+/**
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class RepositoryStack
```

Comment:
I was thinking about interpedent repositories (for example, carsonbot could move the pull requests in the main repository). A stack would allow to fetch data from the sub repo, then change the current repository, then call the github api and then pop the current repository to do other threatments. We can also add this later it doesn't matter too much.